doc/security.md

```diff
@@ -19,10 +19,13 @@ they're launched. The server will use that for all https connections to
 the LXD socket and the client will use its certificate as a client
 certificate for any client-server communication.
 
+To cause certificates to be regenerated, simply remove the old ones, new ones
```

Comment:
"To cause certificates to be regenerated, simply remove the old ones. On the next connection a new certificate will be generated."